logfire/_internal/main.py

```diff
@@ -1828,8 +1842,8 @@ def message_template(self) -> str:  # pragma: no cover
         return ''
 
     @property
-    def tags(self) -> Sequence[str]:  # pragma: no cover
```

Comment:
See the comment on `@message.setter`, tags have the same need, and testing in `test_internal_exception_span`

Reply:
I've added:

```python
    # This is required to make `span.tags = ` not raise an error.
    @tags.setter
    def tags(self, new_tags: Sequence[str]) -> None:
        pass
       
```
        
on the `NoopSpan` class so that setting `span.tags = "bar"` does not raise errors and does nothing. I hope I understood correctly.